packages/sdk/src/cross-chain-messenger.ts

```diff
@@ -351,10 +351,12 @@ export class CrossChainMessenger {
       }
     }
 
+    const minGasLimit = resolved.message.length * 16 + 200_000
```

Comment:
Message here isn't a buffer. You would need to convert to buffer first before getting length.

Reply:
Should be able to use `fromHexString`